### ISSUE TITLE
[Bug] The "language" icon is wrong 

### ISSUE THREAD

**Issue by fhfuih** [tabler/tabler-icons#416]
#396 mentioned this problem, but is off-(that)-topic and after maintainer closed the issue. So I would like to re-mention it here in a new issue.

Note that the following CJK characters can be visually different across machines.
Because each character has regional writing differences, and the default font on your machine to render those character must targets a specific region. More on this later.

## The issue

In the current "language" icon, the CJK Ideograph (Hanzi/Kanji/Hanja) is wrong. It looks more like [丈, which means to measure/the length of 3.3 meter](https://www.unicode.org/cgi-bin/GetUnihanData.pl?codepoint=%E4%B8%88).
It should be [文, which means culture/literature](https://www.unicode.org/cgi-bin/GetUnihanData.pl?codepoint=%E6%96%87).

## Their visual differences
For 丈, there are only three strokes. The last stroke starts from above the horizontal line, going down vertically first, and then all the way to bottom left. (You can see the number of strokes in the two unicode database links above, at the `kTotalStrokes` field.)
For 文, there are four strokes. There is one short curve/line (more on this later) above the horizontal line, and another curve strictly below the horizontal line.

The current source code also follows exactly 丈's strokes, with this single path
```svg
<path d="M9 3v2c0 4.418 -2.239 8 -5 8"></path>
```
going down for 2 and then begin a curve. Hence it is wrong.

## Proofs: 文 instead of 丈

Apart from their (obvious) semantic differences, below are some icons from other products/sets.

![image](https://user-images.githubusercontent.com/11222265/209989742-4af578a5-13d6-4d7b-bdad-8bcd4934cf29.png)
Google Translate from WikiMedia

![image](https://user-images.githubusercontent.com/11222265/209989706-fe4e867d-1870-4704-b5f7-75a7cfa86dc6.png)
Apple Translate, from [PhoneArena](https://www.phonearena.com/news/how-to-use-apple-translate-in-ios_id132673)

## How to write 文, exactly

You can check the regional differences in the 文 character by copy-and-paste this character to the Source Han Sans online demo. It is exactly the same font with different regional differences.
* Simp. Chinese: https://fonts.adobe.com/fonts/source-han-sans-simplified-chinese
* Trad. Chinese: https://fonts.adobe.com/fonts/source-han-sans-traditional-chinese
* Japanese: https://fonts.adobe.com/fonts/source-han-sans-japanese
* Korean: https://fonts.adobe.com/fonts/source-han-sans-korean

Also check out [文's Wikitionary page](https://en.wiktionary.org/wiki/%E6%96%87).
But again, this page's rendering may depend on available fonts.

TLDR: Both Chinese variants have a curve at the top. JP & KR variants have a vertical line above (but at the middle, and it clearly not connected to the other stroke.

Both Chinese variants
<img width="44" alt="image" src="https://user-images.githubusercontent.com/11222265/209990265-87c8af56-76fd-4dcb-bdcb-84f5b98b39e0.png">

JP & KR
<img width="44" alt="image" src="https://user-images.githubusercontent.com/11222265/209990298-98fe4e1c-2765-486b-a581-a07202a069d3.png">

## How to draw a correct icon for this iconset?

The easiest way is to follow the Chinese variants and draw a curve above (and of course change the `M9 3v2` to `M9 5` in the wrong stroke), becase
* The majority of "language"/"translation" icons seems to use the Chinese variant. 
* Tabler has already offered Japanese and Korean versions of "language" (the non-ideograph ones)

But if the maintainers really want inclusion, I suggest replacing the upper stroke with a dot. This vaguely represents either a curve or a vertical line, and serves additional "styling" purpose. The entire iconset is rounded/welcoming/cute. So that should fit.

## More examples of Chinese-variant-文

<img width="106" alt="image" src="https://user-images.githubusercontent.com/11222265/209991874-17ab7de5-ce88-4f41-a553-28beafce66be.png">
Wikitionary

**Comment by fhfuih** [comment]
Since this issue has not been resolved in more than two years (wow...) I just made a PR myself. @codecalm @BG-Software-BG please review and consider adopt it.

I'd like to stress that ideographs work completely differently from phonogram (latin letters, hiragana, katakana, etc.) It is not like you put an "A" to represent the concept of English but now you use "B". It is more like your "AD" icon but with "AO" letters on it, or a "STOP" sign with "FLOP" on it. You just get the wrong meaning at the first sight. And that should have deserved a high severity in an icon set.
